ws-nextjs-app/cypress/e2e/storyPage/index.cy.ts

```diff
@@ -233,3 +298,10 @@ runTestsForPage({
     // ...liteTestSuites
```

Comment:
Do we need the `liteTestSuites`? Can remove this commented code if its no longer needed.